spec/unit/actions/synchronize_user_filters_spec.rb

```diff
@@ -162,11 +161,39 @@
       }
       GoodData::LCM2.convert_to_smart_hash(params)
     end
-
     it 'fails if the MUF set is empty' do
       expect(File).to receive(:open)
       expect(CSV).to receive(:foreach)
       expect { subject.class.call(params) }.to raise_error(/The filter set can not be empty/)
     end
   end
+  context 'when using unsuported sync_mode' do
+    let(:params) do
```

Comment:
Similar code found in synchronize_user_filters_spec.rb:94, synchronize_user_filters_spec.rb:171 (mass = 52)